src/actions/setAirplaneMode.ts

```diff
@@ -0,0 +1,25 @@
+/** @module actions */
+
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+type SetAirplaneModeOptions = {
+  value?: boolean;
+};
+
+/**
+ * Set Airplane Mode Action. Sets the device's Airplane Mode to on or off.
+ * @param {Object} options
+ * @param {boolean} [options.value=true]
+ */
+const setLowPowerMode = (
```

Comment:
Can this be changed to setAirplaneMode please, along with the export?

(I've lost track of the number of times I've done this when adding new actions 🙃!)

Reply:
Haha, classic copy pasta. Will update.